### PR TITLE
swift_lang, fix some wrong requirements

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Made changes to use llvm9 and python3.7 for it, I can't test a build on 32bit, so would be nice if someone could check this out on 64bit
Needs to wait for that and a merge for newer libedit: https://github.com/haikuports/haikuports/pull/5860

**Comment by korli** [comment]
@return could you have a look?

File: dev-lang/swift_lang/swift_lang-4.1.0~git.recipe [hunk 1]
```diff
@@ -130,15 +130,19 @@ BUILD()
 	chmod +x ./swift/utils/build-script ./swift/utils/build-script-impl
 	chmod -R 755 ./swift/utils
 
-	export HOST_CC="/bin/clang"
+	export HOST_CC="/bin/clang_9"
 	export HOST_CXX="/bin/clang++"
 
 	# Build faster by building only for X86
-	python2 ./swift/utils/build-script -R -m \
+	python3.7 ./swift/utils/build-script -R -m \
```
Comment (korli) [inline dev-lang/swift_lang/swift_lang-4.1.0~git.recipe:137]: is it compatible with python3?

Comment (Begasus) [reply dev-lang/swift_lang/swift_lang-4.1.0~git.recipe:137]: Should be investigated, otherwise this PR could be closed, nothing is enabled so far ...

Comment (korli) [reply dev-lang/swift_lang/swift_lang-4.1.0~git.recipe:137]: I see upstream just recently switched to python3.

Comment (Begasus) [reply dev-lang/swift_lang/swift_lang-4.1.0~git.recipe:137]: I could try to switch to the latest version and see if the patchset still applies, but this is one build to start in the evening and hopefully will be done by the morning :)

**Pull request closed without merging**